examples/platform/linux/Options.h

```diff
@@ -53,6 +56,7 @@ struct LinuxDeviceOptions
     chip::Inet::InterfaceId interfaceId = chip::Inet::InterfaceId::Null();
     bool traceStreamToLogEnabled        = false;
     chip::Optional<std::string> traceStreamFilename;
+    chip::Credentials::DeviceAttestationCredentialsProvider * dacProvider = chip::Credentials::Examples::GetExampleDACProvider();
```

Comment:
This is a global static initializer using a singleton getter. This is risky. Please replace with nullptr and initialize it in options.cpp

Reply:
Fixed in the latest commit